app/grpc.go

```diff
@@ -2,12 +2,15 @@ package app
 
 import (
 	"context"
+	"hord/config"
```

Comment:
you'll need to prepend this with `github.com/madflojo/` in order for this to build properly.

Reply:
ok , i'll fix it